setup.cfg

```diff
@@ -2,3 +2,6 @@
 exclude = .git/,.eggs/,build/,dist/,env/,venv/
 max-complexity = 10
 max-line-length = 120
+
+[wheel]
+universal = 1
```

Comment:
I'm new to wheels, so excuse what's possibly a dumb question; is this package 'universal' when it relies on django?

Reply:
universal means that the package isn't tied strictly to a platform architecture (i.e. if built on mac, can run happily on linux). it's true for these libs cos they have no native code